### PR TITLE
Fixes for va-alert-expandable

### PR DESCRIPTION
**Pull request by andresriveratoro**
## Chromatic

https://904-expandable--60f9b557105290003b387cd5.chromatic.com

## Description
Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/904

## Screenshots

va-alert-expandable:
Screen Shot 2022-08-11 at 16 48 22

Screen Shot 2022-08-10 at 10 30 42

va-additional-info:
Screen Shot 2022-08-11 at 16 47 37

## Acceptance criteria
- [x] va-alert-expandable carat does not wrap
- [x] va-alert-expandable trigger text does not wrap below status icon
- [x] va-alert-expandable handles margin for slotted children 

## Definition of done
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Comment by k80bowman** [comment]
could you please add some screenshots?

**Comment by andresriveratoro** [comment]
@k80bowman Updated with screenshots. Sorry about that! I took the liberty of applying the fix for the va-additional-info component, but I can revert it if we don't want that as part of this PR.

**Comment by andresriveratoro** [comment]
@GnatalieH With these changes to va-additional-info, Sketch will have to be updated to exclude the border being applied to the carat. Thanks!

**Comment by k80bowman** [comment]
@andresriveratoro thank you for applying the fix to the additional info component, we definitely want to keep that behavior in sync.

@GnatalieH are you OK with the design changes to this components with the carat? From Andrés's screenshots it looks like the carat is positioned a bit differently and I want to make sure that is all good from a design perspective.

**Review by bkjohnson — COMMENTED** [review]
Are these changes all for items 1, 2 and 4 in the issue?

File: packages/web-components/src/components/va-alert-expandable/va-alert-expandable.tsx [hunk 1]
```diff
@@ -124,10 +124,12 @@ export class VaAlertExpandable {
             class="alert-expandable-trigger"
           >
             <i class="alert-status-icon" aria-hidden="true" role="img"></i>
-            <span class="alert-expandable-title">
-              <span class="sr-only">Alert:&nbsp;</span>
-              {this.trigger}
-            </span>
+            <div>
```
[the remark this answers is not in the document]
Comment (andresriveratoro) [reply packages/web-components/src/components/va-alert-expandable/va-alert-expandable.tsx:127]: This is how it looks like without the container:
Screen Shot 2022-08-11 at 00 19 01

w/ the div container:
Screen Shot 2022-08-11 at 00 16 19

File: packages/web-components/src/components/va-alert-expandable/va-alert-expandable.css [hunk 2]
```diff
@@ -101,9 +103,10 @@ a[aria-expanded='true'] i.fa-angle-down {
   transition: transform 0.15s linear;
 }
 
-i.alert-status-icon { 
+i.alert-status-icon {
   color: var(--color-gray-dark);
-  margin-right: 1.2rem; 
+  margin-top: 0.4rem;
```
[the remark this answers is not in the document]
Comment (andresriveratoro) [reply packages/web-components/src/components/va-alert-expandable/va-alert-expandable.css:108]: When converting the anchor to a flex container, the position of the icon shifted upwards slightly. This moves the icon back to 16px from the border of the component.

[the remark this answers is not in the document]
Comment (andresriveratoro) [reply packages/web-components/src/components/va-alert-expandable/va-alert-expandable.css:108]: All of these changes are best viewed using the smallest breakpoint in Storybook and with a long enough alert headline to produce wrapping.

**Comment by GnatalieH** [comment]
@andresriveratoro and @k80bowman it looks like the Additional info carat is too far to the right of the text. It should remain in the top right position, but needs to have some righthand padding and remain close enough to the text that it is identifiable as being connected to the underlined text.

**Comment by andresriveratoro** [comment]
@GnatalieH The reason the carat appears too far is because the alert headline text wrapped to the next line. This is how it looks like when the text doesn't wrap:

Screen Shot 2022-08-11 at 13 47 33

**Comment by GnatalieH** [comment]
@andresriveratoro I think the alert looks good, it's the Additional info carat that looks strange to me. Thx.

**Comment by andresriveratoro** [comment]
@GnatalieH Sorry! I posted the wrong screenshots but the behavior here is the same. Do we want the carat to follow the last character of the text or do we want the carat positioned at the top right? If we want it positioned at the top right, there will be times where the spacing will look like a lot depending on how the text wraps.

**Comment by GnatalieH** [comment]
Thanks for those examples @andresriveratoro! I think our better option is to have the carat follow the last character of the text. Thanks so much for trying out the different treatments and including screenshots!

File: packages/web-components/src/components/va-alert-expandable/va-alert-expandable.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (bkjohnson) [reply packages/web-components/src/components/va-alert-expandable/va-alert-expandable.tsx:127]: This makes total sense, good job :+1:

**Review by bkjohnson — APPROVED** [review]
Looks good, thank you for the explanations.

**Comment by andresriveratoro** [comment]
The carat was updated to be positioned inline after the text instead of remaining top right when the text wraps. Please see the screenshots included in the PR description.

**Pull request merged**